src/System.Reflection.Emit/tests/AssemblyBuilder/AssemblyBuilderSetCustomAttribute.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using System.Reflection;
+using System.Reflection.Emit;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    [AttributeUsage(AttributeTargets.All, AllowMultiple = false)]
+    public class ABAttribute1 : Attribute
+    {
+        private bool _s;
+
+        public ABAttribute1(bool s)
+        {
+            _s = s;
+        }
+    }
+
+    public class AssemblyBuilderSetCustomAttribute1
+    {
+        [Fact]
+        public void PosTest1()
+        {
+            AssemblyBuilder asmBuilder = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName("PT1"), AssemblyBuilderAccess.Run);
+            ConstructorInfo c = typeof(ABAttribute1).GetConstructor(new Type[] { typeof(bool) });
+            asmBuilder.SetCustomAttribute(c, new byte[] { 01, 00, 01 });
```

Comment:
I am testing this by checking the set custom attribute. github is not outdating this diff.